### ISSUE TITLE
8*MLP was trained during finetune

### ISSUE THREAD

**Issue by cnnlstm** [HideUnderBush/UI2I_via_StyleGAN2#3]
Thanks for your excellent work, but it seems that when finetuning the model on a new domain data, the mapping net(8*mlp) was not frozen, which conflicts with your papers, though requires_grad==False was set in L422-425 of train.py. The gradient is activated in L229 again and the G_optimizer optimize all parameters of G. When I print the parameters of 8*MLP on the original model and finetuned, they indeed different.

**Comment by HideUnderBush** [comment]
Thanks for pointing it out, there might be some missing lines when migrating the code from server to public repo. We'll look into it and update code soon.
